department-of-veterans-affairs/va.gov-cms
# VACMS-19772: update composer dependencies first round.

## Description
This PR encapsulates the work for updating modules and exporting configurations related. A full list of the updates are here: https://docs.google.com/spreadsheets/d/1exMktEjiY9CaKtzvh0Qi-F8ffuDz-jJ5LXMuHVx4cu0/edit?usp=sharing

Relates to #19772 

## Testing done
Local logs and spot checking.

## QA steps

What needs to be checked to prove this works?
Look at list of composer modules updated to see if it matches expectations.
What needs to be checked to prove it didn't break any related things?
Site shouldn't change any functionality or look.
What variations of circumstances (users, actions, values) need to be checked?

### Definition of Done


### Select Team for PR review
